static/src/javascripts/projects/common/modules/commercial/front-large.js

```diff
@@ -0,0 +1,35 @@
+define([
+    'bonzo',
+    'common/utils/$',
+    'common/utils/config',
+    'common/modules/commercial/create-ad-slot'
+], function (
+    bonzo,
+    $,
+    config,
+    createAdSlot
+) {
+
+    function init() {
+        if (typeof config.tests.topBannerPosition === 'undefined' || !config.page.isFront) {
+            return false;
+        }
+
+        var containerIndex = 0,
+            $adSlotWrapper = $.create('<div class="fc-container large-top top-banner-ad-container top-banner-ad-container--desktop top-banner-ad-container--above-nav"></div>'),
```

Comment:
I'm not sure it's obvious what a `large-top` is. It seems to imply a larger upper banner, perhaps?